uaclient/entitlements/tests/test_base.py

```diff
@@ -88,6 +89,7 @@ def factory(
             applicability_status=applicability_status,
             application_status=application_status,
             allow_beta=allow_beta,
+            enable=enable,
```

Comment:
Why do we need this variable for the new test ?

Reply:
@lucasmoura This variable is used as the return value of `_perform_enable` for `ConcreteTestEntitlement`.

The new/modified test calls `entitlement.enable()` with the expectation that it reaches the point of calling `self._perform_enable()` after it disables the incompatible services.

Since we expect the code to call `_perform_enable`, we should set the return value for `_perform_enable` in the concrete test entitlement via this argument. I'll add a comment where it is used.

Reply:
My mistake here, I missed the `_perform_enable` override